R/with_opt.R

```diff
@@ -53,6 +53,43 @@ with_golem_options <- function(
 #' @param which NULL (default), or the name of an option
 #' @importFrom shiny getShinyOption
 #' @export
+#' @examples 
+#' 
+#' \dontrun{
+#' # 1. Pass parameters to `run_app`
+#' 
+#' run_app(p1 = "param1", p2 = "param2")
+#' 
+#' # 2. Use get_golem_options()
+#' 
+#' ## in UI...
+#' 
+#' h1(get_golem_options("p1"))
+#' 
+#' ## ...or in server
+#' 
+#' output$param <- renderPrint({
+#'   paste("param p2 = ",get_golem_options("p2"))
+#' })
+#' 
+#' # Et voila.
+#' 
+#' # to set default value, edit run_app like this :
+#' run_app <- function(
+#' ..., p3 = "default p3"
```

Comment:
Do you see any good reason for leaving the `...` here? 

I don't have a use case in mind where we would let the user enter unpredicted arguments?

Reply:
This is the default behavior...

Reply:
This is the default because the dev needs to __fill it__. 
There would be no reason for the person calling `run_app()` to fill `...` as it would be ignored by the app.